spec/models/patient_spec.rb

```diff
@@ -76,49 +82,191 @@
   end
 
   context 'Scopes' do
-    describe '.follow_ups' do
-      let!(:current_user) { create(:user) }
-      let!(:current_facility) { create(:facility, facility_group: current_user.facility.facility_group) }
-      let!(:reg_date) { Date.new(2018, 1, 1) }
-      let!(:patient_with_follow_up) { create(:patient, registration_facility: current_facility, recorded_at: reg_date) }
-      let!(:patient_without_follow_up) { create(:patient, recorded_at: reg_date) }
+    describe '.diabetes_only' do
+      it 'only includes patients with diagnosis of diabetes' do
+        create_list(:patient, 2, :diabetes)
+        create(:patient, :hypertension)
+
+        expect(Patient.diabetes_only.count).to eq(2)
+      end
+    end
+
+    describe '.hypertension_only' do
+      it 'only  includes patients with diagnosis of hypertension' do
+        create_list(:patient, 2, :hypertension)
+        create(:patient, :diabetes)
+
+        expect(Patient.hypertension_only.count).to eq(2)
```

Comment:
This would be a better spec if you asserted the records directly rather than a count with `match_array`.